preview/MsixCore/msixmgr/InstallUI.cpp

```diff
@@ -45,10 +46,11 @@ HRESULT UI::DrawPackageInfo(HWND hWnd, RECT windowRect)
     }
     else
     {
+        auto displayText = m_displayName + L" " + GetStringResource(IDS_STRING_LOADING_PACKAGE_ERROR);

```

Comment:
>m_displayName [](start = 27, length = 13)

what about cases where we can't even open the package?

Reply:
I am implementing it for 0x8bad0031 (MissingAppxSignatureP7X),
0x8bad0042 (CertNotTrusted)  and
0x8bad0001 (FileOpen)